_sass/_components/_logo-bar.scss

```diff
@@ -101,11 +82,17 @@ $logo-bar-scale: 14px;
           max-height: $logo-bar-scale * 1.25;
         }
 
-        &[src*="zipkin_small.png"] {
+        &[src*="grpc"] {
```

Comment:
@bensigelman making 'supported tracers' clickable + fixing this would go well together. My initial implementation assumed images, but the introduction of text-only made things a little bit...quirky. 

So some css changes + turning the pseudo-classes into actual html is my proposition here. Should be fairly quick. :smile: